### ISSUE TITLE
update docs on play/pause sound component

### ISSUE THREAD

**Issue by rishitank** [aframevr/aframe#1916]
**Description:**
When calling pause() on a sound component, the sound pauses but it feels as if the counter for the sound playback is still ongoing (it is as if pause() just mutes the volume) and calling play() does not resume the audio where it left off, but instead however many X seconds have passed since the initial pause().

- A-Frame Version: 0.3.0
- Platform/Device: Desktop Chrome 53.0.2785.116 m
- Reproducible Code Snippet or Demo URL [highly encouraged]:

HTML:
`<a-entity id="soundEntity" sound="src: #black; on: mouseenter" position="0 5.2 0.3" scale="0.2 0.2 0.2"></a-entity>`

JavaScript:
```
var soundEntity = document.querySelector("#soundEntity");
soundEntity.addEventListener("stateremoved", function(evt) {
    if (evt.detail.state === "cursor-hovered" && soundEntity.components.sound.isPlaying) {
        soundEntity.components.sound.pause();
    }
});
soundEntity.addEventListener("stateadded", function(evt) {
    if (evt.detail.state === "cursor-hovered" && !soundEntity.components.sound.isPlaying) {
        soundEntity.components.sound.play();
    }
});
```

Also if I were to log `soundEntity.components.sound.sound.context.currentTime` after a call to pause or play, the counter has a gap of however many seconds I wait between each pause and play. Any help with this would be greatly appreciated, thanks.

**Comment by ngokevin** [comment]
Try `play/pauseSound()` or pause/play the audio element

**Comment by rishitank** [comment]
Thanks, pausing/playing directly on the HTML5 audio element works as expected.

**Comment by ngokevin** [comment]
Cool, changing the issue to improve our docs.

**Comment by cvan** [comment]
Perhaps this is deserving of its own issue, and let me know, and I can file.

When I was dealing with some scenes in the past, I wanted to just mute all sounds on all the entities in the scene. [Here was my hacky solution.](https://github.com/MozVR/oculus/blob/e73ea2c8/assets/js/common.js#L20-L85)

If there's already a component that handles this, awesome! If not, can we introduce it into core (e.g., `AFRAME.sounds.mute()` or something)? Or, if folks feel it should be a third-party component (even one under the `aframevr` org), what would it look like? (I assume my [approach](https://github.com/MozVR/oculus/blob/e73ea2c8/assets/js/common.js#L20-L85) above is not a good one to evangelize. I kept hitting issues, and that hacked-up mess is what I arrived at.)

I see this issue is being repurposed to improve the docs. Regardless of the componentisation/functionality point above, can we also document how to pause all sounds in a scene?
